lib/database/seeders/20240404100811-qc-flags.js

```diff
@@ -0,0 +1,134 @@
+/**
+ *  @license
+ *  Copyright CERN and copyright holders of ALICE O2. This software is
+ *  distributed under the terms of the GNU General Public License v3 (GPL
+ *  Version 3), copied verbatim in the file "COPYING".
+ *
+ *  See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ *  In applying this license CERN does not waive the privileges and immunities
+ *  granted to it by virtue of its status as an Intergovernmental Organization
+ *  or submit itself to any jurisdiction.
+ */
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+    up: async (queryInterface) =>
+        queryInterface.sequelize.transaction((transaction) =>
+            Promise.all([
+                queryInterface.bulkInsert('quality_control_flags', [
+
+                    /** Flags for runNumber: 106, LHC22b_apass1, CPV */
+                    {
+                        id: 1,
+                        from: new Date((1565314200 - 10000) * 1000),
+                        to: new Date((1565314200 + 10000) * 1000),
```

Comment:
Please use date like "2023-01-01T01:01:01", it will be way more clear when writing tests

Reply:
applied